### DISCUSSION TITLE
Title of note showing as title in dashboard

### DISCUSSION THREAD

**Discussion by alexander-meyenberg** [TfTHacker/DashboardPlusPlus#33] [Q&A]
As the title suggests, I have used the CSS snippets to create a dashboard. I have changed the banner size from 150px to 300px in the banner-image.css file (found file here: https://tfthacker.com/banner-image). I have not changed any other code, and would like to find a way to remove the header from my dashboard and keep it only at the top. 

![Screenshot 2024-08-21 at 2 21 54 PM](https://github.com/user-attachments/assets/e5ee3037-32c9-4ac9-a292-93cab06b3d09)
![Screenshot 2024-08-21 at 2 22 00 PM](https://github.com/user-attachments/assets/d0d5a16d-a93a-49b1-a9af-5e3c2e2fdc4d)
![Screenshot 2024-08-21 at 2 22 21 PM](https://github.com/user-attachments/assets/fe3066d0-c107-4ee6-b322-0d7ce7320249)

**Comment by TfTHacker** [comment]
Did you implement your header with an html div? this moves it up into the banner area.

[https://tfthacker.com/article-obsidian-dashboardplusplus2022#Adding+a+page+title](https://tfthacker.com/article-obsidian-dashboardplusplus2022#Adding+a+page+title)
